pkg/config/render_config.go

```diff
@@ -92,6 +93,13 @@ func mkContext(buildType string) context {
 			KubeApiServer: true,
 			ClusterChecks: true,
 		}
+	case "dcacf":
```

Comment:
What is `dcacf `?

Reply:
It would be abbreviation of DCA for Cloud Foundry; it's only used here for now, but when we have also git tags for releasing, we would be using it also in the new tasks file that's also part of this PR (can't do that now as no tag exists with this name yet).